api/v1/merge_strategies.go

```diff
@@ -32,6 +32,7 @@ const (
 	PolicyRuleMergeStrategy = "merge"
 )
 
+// +kubebuilder:object:generate=false
```

Comment:
@guicassolato Had to exclude this due to this error in generate:
```
merge_strategies.go:36:2: invalid field type: any
Error: not all generators ran successfully
```

I don't think it needs to be included anyway, let me know if you think otherwise.